docs/components/choice-tiles/test.html

```diff
@@ -0,0 +1,500 @@
+---
```

Comment:
Is the `test.html` file a new thing? Will/Should this appear on the site, or is it for QE?

Reply:
good question! is it for QE -- we found the states to be a little overwhelming for the component/demo page but necessary for testing/ making sure that the choice tiles reflected the design spec states

Reply:
Ok, so this isn't necessarily needing to be linked anywhere for discovery on the site. Got it.

Reply:
Correct. `test.html` files are going to be the home for any toolkit developer testing (visual regression, client-side framework compatibility, etc.)